src/components/Tooltip/__snapshots__/index.test.js.snap

```diff
@@ -5,82 +5,55 @@ exports[`Tooltip component matches snapshot (uses the right classes) 1`] = `
   message="lol"
   trigger={<Trigger />}
 >
-  <Popover
-    alignment="center"
-    delay={350}
-    distance={8}
-    interactionMode="hover"
-    position="bottom"
-    transitionName="GrowFast"
-    trigger={
-      <span
-        style={
-          Object {
-            "cursor": "help",
-          }
-        }
-      >
-        <Trigger />
-      </span>
+  <ForwardRef(TippyWrapper)
+    appendTo={<body />}
+    arrow={false}
+    className="bgColor--charcoal inverted align--center padding--top--half padding--bottom--half padding--left padding--right elevation--2 rounded--all fontSize--s fontWeight--bold"
+    content="lol"
+    delay={
```

Comment:
One requested feature we had for Tooltip previously was a delay.

I feel pretty strongly that the delay should **not** be configurable; it should be an on or off option so that we can make hover delays consistent between components.

Would it be difficult to add a `hasDelay` prop that defaults to true?

Reply:
What do you think about having that be a separate task / ticket as a followup? It wouldn't be difficult in either case.

I'm with you on not having delay configurable and just relying on a boolean in the facade.

Reply:
resolving with follow up ticket https://cbinsights.atlassian.net/browse/UIP-1008